src/v2/ion/ui-schema/ion-object-handler.js

```diff
@@ -10,12 +10,46 @@
  * See the License for the specific language governing permissions and limitations under the License.
  */
 
+import { _ } from 'okta';
+
 /**
  * Example of the option like
  * @param {AuthenticatorOption[]} options
  * @param {( AuthenticatorEnrollment[] || Authenticator[] )} authenticators
  */
+const createOVOptions = (options = []) => {
+  // Split OV into individual entries for verification (one for each method).
+  const ovItem = options.find((option) => option.relatesTo.type === 'app');
+  const methodTypeObj = ovItem?.value?.form?.value?.find(v => v.name === 'methodType');
```

Comment:
Just to double check  this doesnt break on .find because of optional chaining ?

Reply:
Checking now. My understanding is it would return if value is null or undefined.. So it would never get to `.find`.

Reply:
Tested this by removing `value` from `ovItem.value.form.value`, still works 👍